### DISCUSSION TITLE
Small project to try out Csla 8.xx

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3716] [Questions]
I have been asked to develop a small project (internal). I would like to try out the newest version of Csla. The UI doesn't matter, I could use the good old Winforms : ).
Do we have any sample code?  I think I would be needing Edit Root collection, at least, for the time being.

Kind Regards

**Comment by rockfordlhotka** [comment]
If the UI doesn't matter, I'd suggest trying Blazor. If you look at the open PR where I'm building out the updated ProjectTracker you can see how that's coming together.

However, all the other (simpler) examples should be working today - WinFormsExample, RazorPagesExample, WpfExample, etc.
